tutorials/Tutorial6_Better_Retrieval_via_EmbeddingRetrieval.ipynb

```diff
@@ -0,0 +1,446 @@
+    "\n",
+    "\n",
+    "\n",
+    "##### FARMReader"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "fyIuWVwhA6OB"
+   },
+   "outputs": [],
+   "source": [
+    "# Load a  local model or any of the QA models on\n",
+    "# Hugging Face's model hub (https://huggingface.co/models)\n",
+    "\n",
+    "reader = FARMReader(model_name_or_path=\"deepset/roberta-base-squad2\", use_gpu=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "unhLD18yA6OF"
+   },
+   "source": [
+    "### Pipeline\n",
+    "\n",
+    "With a Haystack `Pipeline` you can stick together your building blocks to a search pipeline.\n",
+    "Under the hood, `Pipelines` are Directed Acyclic Graphs (DAGs) that you can easily customize for your own use cases.\n",
+    "To speed things up, Haystack also comes with a few predefined Pipelines. One of them is the `ExtractiveQAPipeline` that combines a retriever and a reader to answer our questions.\n",
+    "You can learn more about `Pipelines` in the [docs](https://haystack.deepset.ai/docs/latest/pipelinesmd)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "TssPQyzWA6OG"
+   },
+   "outputs": [],
+   "source": [
+    "from haystack.pipelines import ExtractiveQAPipeline\n",
+    "\n",
+    "pipe = ExtractiveQAPipeline(reader, retriever)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "bXlBBxKXA6OL"
+   },
+   "source": [
+    "## Voilà! Ask a question!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "Zi97Hif2A6OM"
+   },
+   "outputs": [],
+   "source": [
+    "# You can configure how many candidates the reader and retriever shall return\n",
+    "# The higher top_k for retriever, the better (but also the slower) your answers.\n",
+    "prediction = pipe.run(\n",
+    "    query=\"Who created the Dothraki vocabulary?\", params={\"Retriever\": {\"top_k\": 10}, \"Reader\": {\"top_k\": 5}}\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "pI0wrHylzqLa"
+   },
+   "outputs": [],
+   "source": [
+    "print_answers(prediction, details=\"minimum\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "kXE84-2_zqLa"
+   },
+   "source": [
+    "## About us\n",
+    "\n",
+    "This [Haystack](https://github.com/deepset-ai/haystack/) notebook was made with love by [deepset](https://deepset.ai/) in Berlin, Germany\n",
+    "\n",
+    "We bring NLP to the industry via open source!  \n",
+    "Our focus: Industry specific language models & large scale QA systems.  \n",
+    "  \n",
+    "Some of our other work: \n",
+    "- [German BERT](https://deepset.ai/german-bert)\n",
+    "- [GermanQuAD and GermanDPR](https://deepset.ai/germanquad)\n",
+    "- [FARM](https://github.com/deepset-ai/FARM)\n",
```

Comment:
@mkkuemmel @sjrl:  Maybe we should stop advertising FARM in the tutorials? :thinking:  Just an idea, idk what's the policy here.

Reply:
We definitely should discuss this at some point and find a policy for it! For now, I'd leave it in there and either change it for every tutorial later or not at all.

Reply:
Especially as we're still using FarmReader

Reply:
@mkkuemmel Right, but the FarmReader was moved into Haystack so it might actually be confusing that there is a FarmReader in both Haystack and in FARM.

Reply:
Ah I see, that's a good point. I'd still suggest to discuss this with e.g. Timo (as one of the creators of FARM) and then remove it from all of the tutorials at the same time, what do you think?

Reply:
I agree that sounds like the right way to go.